### ISSUE TITLE
Internal Dirac Error: Cannot read property 'text' of undefined

### ISSUE THREAD

**Issue by superstructor** [binaryage/dirac#32]
```
[binaryage/dirac "0.6.3"]
[binaryage/devtools "0.8.1"]
[powerlaces/boot-cljs-devtools "0.1.1"]
```

Chrome `54.0.2836.0 canary` on macOS.

On loading the page the normal CLJS DevTools / Dirac Runtime installation messages are printed in the console, then shortly after the following exception is printed in the console. CLJS REPL still appears to be working regardless.

```
Internal Dirac Error: DevTools code has thrown an unhandled exception
TypeError: Cannot read property 'text' of undefined
    at WebInspector.RuntimeDispatcher.exceptionThrown (chrome-extension://kbkdngfljkchidcjpnfcgcokkbhlkogi/devtools/front_end/inspector.js:6309:180)
    at Object.dispatch (chrome-extension://kbkdngfljkchidcjpnfcgcokkbhlkogi/devtools/front_end/inspector.js:4355:63)
    at WebInspector.WebSocketConnection.dispatch (chrome-extension://kbkdngfljkchidcjpnfcgcokkbhlkogi/devtools/front_end/inspector.js:4295:31)
    at WebInspector.WebSocketConnection._onMessage (chrome-extension://kbkdngfljkchidcjpnfcgcokkbhlkogi/devtools/front_end/inspector.js:10993:31)
```

**Comment by pepe** [comment]
I have same issue, only not on Page Load, but when any exception is thrown by my `re-frame` application.

Same dependencies as @superstructor, Chrome `54.0.2837.0 canary` on OS X `10.11.6 `.

Output in the vanila dev tools:
`Error rendering component (in mtrack_planner.views.main_panel)`
 `Uncaught Error: Oops`

Output in the Dirac dev tools:
`Error rendering component (in mtrack_planner.views.main_panel)`
```
Internal Dirac Error: DevTools code has thrown an unhandled exception
TypeError: Cannot read property 'text' of undefined
    at WebInspector.RuntimeDispatcher.exceptionThrown (chrome-extension://kbkdngfljkchidcjpnfcgcokkbhlkogi/devtools/front_end/inspector.js:6309:180)
    at Object.dispatch (chrome-extension://kbkdngfljkchidcjpnfcgcokkbhlkogi/devtools/front_end/inspector.js:4355:63)
    at WebInspector.WebSocketConnection.dispatch (chrome-extension://kbkdngfljkchidcjpnfcgcokkbhlkogi/devtools/front_end/inspector.js:4295:31)
    at WebInspector.WebSocketConnection._onMessage (chrome-extension://kbkdngfljkchidcjpnfcgcokkbhlkogi/devtools/front_end/inspector.js:10993:31)
```

**Comment by superstructor** [comment]
The application for which I have this issue is also using `[re-frame 0.8.0]`.

**Comment by pepe** [comment]
Thanks for mentioning. For me it is `[re-frame 0.7.0]`

**Comment by darwin** [comment]
This is probably caused by new Canary version not matching older DevTools included in Dirac v0.6.3.

I'm about to release Dirac v0.6.4 which will likely fix it. Will get to it on Thursday.

**Comment by darwin** [comment]
This error should go away after upgrading to [v0.6.4](https://github.com/binaryage/dirac/releases/tag/v0.6.4).

Please reopen if not.
